src/fields/core/fieldInput.vue

```diff
@@ -6,6 +6,7 @@
 		:value="value",
 		@input="onInput",
 		@blur="onBlur",
+		@keyup.enter="onDataAccept",
```

Comment:
This should go through the same path as `blur` which is just flushing out any debouncing that is being done.

```suggestion
		@keyup.enter="onBlur",
```